src/CoreShop/Bundle/ProductBundle/DependencyInjection/Configuration.php

```diff
@@ -228,14 +228,20 @@ private function addPimcoreResourcesSection(ArrayNodeDefinition $node)
                             ->scalarNode('product')->defaultValue('/bundles/coreshopproduct/pimcore/css/product.css')->end()
                         ->end()
                     ->end()
-                    ->scalarNode('permissions')
-                        ->cannotBeOverwritten()
-                        ->defaultValue(['product_price_rule'])
+                    ->arrayNode('permissions')
```

Comment:
don't overwrite permission configurations, you cannot change them anyway and its not part of this PR